revscoring/languages/__init__.py

```diff
@@ -1,3 +1,46 @@
+"""
+This module implements a set of :class:`~revscoring.languages.language.Language`
+for use in the extraction of language-dependent features.  A language generally
+implements 4 :class:`~revscoring.languages.language.LanguageUtility`:
+
+* :data:`~revscoring.languages.language.stem_word`
+* :data:`~revscoring.languages.language.is_badword`
+* :data:`~revscoring.languages.language.is_misspelled`
+* :data:`~revscoring.languages.language.is_stopword`
+
+A growing set of :class:`~revscoring.languages.language.Language` are provided
+as part of the common installation.
+
+english
++++++++
+.. automodule:: revscoring.languages.english
```

Comment:
I don't know if these are expected, but for each language we have something like this:
`<revscoring.languages.language.Language object at 0x2b263b8258d0>`
at http://pythonhosted.org/revscoring/revscoring.languages.html